rosidl_generator_dds_idl/rosidl_generator_dds_idl/__init__.py

```diff
@@ -52,11 +55,76 @@ def generate_dds_idl(
                     },
                     globals=data,
                 )
-                interpreter.file(open(template_file))
+                interpreter.file(open(template_file_msg))
                 interpreter.shutdown()
             except Exception:
                 os.remove(generated_file)
                 raise
+        elif extension == '.srv':
+            srv_spec = parse_service_file(pkg_name, idl_file)
+            default_value_string = None
+            request_fields = [
+                Field(
+                    Type('int64', context_package_name=pkg_name),
+                    'sequence_id', default_value_string),
```

Comment:
Please always use the explicit keyword for kwargs (instead of relying on the position). This will also make the local variable `default_value_string` obsolete.